playbooks/appsemblerPlaybooks/eucalyptus_pro_server.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+# Appsembler pro tier deployment
+# Docs: TO-DO
+
+- name: Configure mongo
+  hosts: "mongo-server"
+  sudo: True
+  gather_facts: True
+  vars:
+    BACKUPS_MONGO: yes
+    BACKUPS_MYSQL: no
+    COMMON_ENABLE_BACKUPS: True
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
```

Comment:
@johnbaldwin If Mongo is using swap, we need to upgrade the machine because it kills performance.